_posts/2024-06-30-update.md

```diff
@@ -7,11 +7,17 @@ slug: 2024-06-30-update
 
 ## Developer News
 
+The [Kubernetes Contributor Summit](https://www.kubernetes.dev/events/2024/kcsna/) is looking for [day-of-event volunteers](https://forms.gle/YAFo8qJn4P3g7WQQ9).  If you think you'll be going and want to help out, please fill out the form.  Also don't forget to [propose a session](https://docs.google.com/forms/d/e/1FAIpQLSfqdvHnS4HVZQXdBmZHClgUbAodxEGH18t365qqdgtn0hhx-Q/viewform).
+
+Paul Morie will be [stepping down](https://groups.google.com/a/kubernetes.io/g/dev/c/HRjOyF02hyA) from SIG-Multicluster leadership.  Stephen Kitt is nominated to replace him.
+
+[Maintainer Track proposals](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/program/project-opportunities/) for SIGs and WGs are open until July 20 for KubeCon SLC.
```

Comment:
Source: Kubecon announcements, Chairs and TLs meeting.    While we don't usually add Kubecon news to LWKD, the maintainer track proposals get added specifically because it affects our SIG leads.